### PR TITLE
New package: rustfmt-0.6

### PR DESCRIPTION
**Pull request by DirectorX**

### CODE REVIEW

**Comment by DirectorX** [comment]
**rebased from** #4527

File: common/build-style/cargo.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#
+# This helper is for templates for Cargo packages.
+#
+
+do_build() {
+	cargo build --release
+}
+
+do_install() {
+	cargo install
```
Comment (ebfe) [inline common/build-style/cargo.sh]: This installs files into ./target/release.

Comment (DirectorX) [inline common/build-style/cargo.sh]: well that weird, i use rustfmt that i build using those scripts
is probably something has changed, again

Comment (ebfe) [inline common/build-style/cargo.sh]: Sorry, not `./target/release` but `$HOME/.cargo/bin`. And as far as i can tell, that's how `cargo install` always worked.

**Pull request closed without merging**